client/packages/purchasing/src/purchase_order/api/hooks/usePurchaseOrderLine.ts

```diff
@@ -127,6 +127,7 @@ export function usePurchaseOrderLine(id?: string | null) {
       supplierItemCode: setNullableInput('supplierItemCode', draft),
       comment: setNullableInput('comment', draft),
     };
+    resetDraft();
```

Comment:
Without this, if the incoming line (when clicking "Next") didn't already have a value for "Requested" it would get pre-filled with the last value, which wasn't good.